contents/zeromq/zeromq.md

```diff
@@ -0,0 +1,97 @@
+<frontmatter>
+  title: Introduction to ZeroMQ
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# Introduction to ZeroMQ
+
+Author: [Tan Yuanhong](https://github.com/le0tan)
+
+<box id="article-toc">
+
+* [What is ZeroMQ?](#what-is-zeromq)
+  * [What is Asynchronous Messaging and Message Queue?](#what-is-asynchronous-messaging-and-message-queue)
+* [Why ZeroMQ?](#why-zeromq)
+  * [Ease of Use](#ease-of-use)
+  * [Multi-Transport](#multi-transport)
+  * [High Performance](#high-performance)
+* [Getting Started](#getting-started)
+</box>
+
+## What is ZeroMQ?
+
+[ZeroMQ](https://zeromq.org/) is a high-performance asynchronous messaging library. It provides a message queue (MQ) for all processes communicating using the library, without a dedicated message broker - thus the *zero* in the name of *ZeroMQ*. 
+
+### What is Asynchronous Messaging and Message Queue?
+
+Asynchronous (or async, in short) communication means the two (or more than two) ends of the communication don't need to be occupied at the same time to send messages. You may consider making a phone-call synchronous and sending an email async. And **message queue** is one of the async communication protocols that puts the messages in a queue, store them temporarily until the recipient retrieves them.
```

Comment:
Also, the heading is `Asynchronous Messaging` but in the paragraph, you use `Asynchronous communication`. Better to make them similar, so that the reader doesn't get confused.

Depending on whichever way you proceed, please bold it as it will make it similar with **message queue**

Reply:
Maybe you could also point them to some link or site which explains this in greater detail.